package.json

```diff
@@ -57,11 +57,10 @@
     "eslint": "^3.3.1",
     "http-server": "^0.9.0",
     "jsdom": "^9.4.2",
+    "lodash": "^4.15.0",
```

Comment:
Also if we only want a simple throttle/debounce, here's one: https://github.com/vvo/in-viewport/blob/9ca9f9172187b749dc5a4ae8f6f48f1f60fec147/in-viewport.js#L39-L53